dev/a11y_assessments/lib/use_cases/text_field_password.dart

```diff
@@ -29,21 +29,25 @@ class _MainWidget extends StatelessWidget {
         title: const Text('TextField password'),
       ),
       body: ListView(
-        children: const <Widget>[
-          TextField(
-            key: Key('enabled password'),
-            decoration: InputDecoration(
-              labelText: 'Password',
+        children: <Widget>[
+          Semantics(
+            child: const TextField(
+              key: Key('enabled password'),
+              decoration: InputDecoration(
+                labelText: 'Password',
+              ),
+              obscureText: true,
             ),
-            obscureText: true,
           ),
-          TextField(
-            key: Key('disabled password'),
-            decoration: InputDecoration(
-              labelText: 'Password',
+          Semantics(
```

Comment:
I can't wrap my head around this, do you know what exactly does a Semantics wrapper do here?

Reply:
Without the Semantics wrapper, the HTML input element does not have the 'disabled' attribute added. Screen readers need to see this attribute to know to not focus on this element.

https://screenshot.googleplex.com/BkyKJRgNyCJDEuk

Reply:
but i can't see how this could have happened with just an semantics widget. Do you know what code path that lead to flipping the flag with this semantics widget? I would like to know because it feels to me a bug in the code somehow accidentally fixes this original issue with this semantics widget.

Reply:
also cc @yjbanov if you can spot anything that can lead to this

Reply:
After investigating this more, I found that by taking the Semantics wrapper off, the disabled attribute still exists in the localhost demo: 
![image](https://github.com/user-attachments/assets/73b546d0-7046-4e60-b5b7-d12c0b30d4a7)

I'm not sure why this shows in the localhost and not in the https://fwa11y.web.app/#/text-field-password

Is there a configuration missing somewhere that's missing whenever the production app is setup?